commands/releases/output.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+let cli = require('heroku-cli-util')
+let co = require('co')
+
+function * run (context, heroku) {
+  function getRelease (id) {
+    return heroku.get(`/apps/${context.app}/releases/${id}`)
+  }
+
+  function getLatestRelease () {
```

Comment:
do you think we could extract this into a new file and share it with https://github.com/heroku/heroku-apps/blob/master/commands/releases/info.js#L13 ? (and anywhere else we're doing the same)